src/dbSta/src/dbNetwork.cc

```diff
@@ -867,6 +986,17 @@ void dbNetwork::readDbAfter(odb::dbDatabase* db)
       makeLibrary(lib);
     }
     readDbNetlistAfter();
+    // we make the library for the verilog hierarchical cells
+    // this is in the same fashion as the original dbInst code
+    // which uses the void* staGetCell to associate a cell with
+    // concrete cell. We do same for verilog hierarchical cells.
+    Library* verilog_library = makeLibrary("verilog", nullptr);
+    dbSet<dbModInst> modinsts = block_->getModInsts();
+    dbSet<dbModInst>::iterator modinst_iter_ = modinsts.begin();
+    dbSet<dbModInst>::iterator modinst_end_ = modinsts.end();
+    for (; modinst_iter_ != modinst_end_; modinst_iter_++) {
+      makeVerilogCell(verilog_library, *modinst_iter_);
+    }
```

Comment:
Should this happen if you are not running in hierarchical mode?

Reply:
ok, gated on hierarchy_.

Reply:
(note we hope in the next pull request after this one to somehow avoid the weird library creation stuff altogether so it is benign).